### ISSUE TITLE
Description of output files

### ISSUE THREAD

**Issue by khembach** [philippdre/omniCLIP#2]
It would be great if you could describe the output files and especially the different columns in `pred.txt`. Is column `pv` the log10 of the p-value of the peak and what is the `SiteScore`? 
I assume that column `TC` is the number of T->C conversions, but are other diagnostic events, e.g. deletions, also reported?
Thanks.

**Comment by philippdre** [comment]
Yes, pv the log10 of the p-value. 

SiteScore is the log-likelihood ratio of the peak state versus the other states.

The TC column counts the number of T->C conversions.

**Comment by khembach** [comment]
Thank you! Does that mean that only the T->C conversions are reported but not deletions? I am asking, because I work with HITS-CLIP data and I would like to know the number of deletions per peak.

**Comment by philippdre** [comment]
Yes, it is still a legacy from working with PAR-CLIP data. I will try to change it in the next version.

**Comment by khembach** [comment]
Ok thanks, than I will try to find a way to compute the deletions.

**Comment by dataMaster-Kris** [comment]
Have the columns in the output files been described anywhere yet? It will be much appreciated if there was a proper documentation of how to interpret each of the columns in the tables (the above discussion helps with interpreting `pv`, `SiteScore` and `TC` but I am having to figure out the rest on my own). Are the p-values in pred.txt Bonferroni corrected? Also, what is the difference between pred.txt and pred.bed?

**Comment by santataRU** [comment]
I also hope there is documentation for this tool. I do not know what the term “dir_score” means in the pred.txt file in the output folder. My pred.bed file contains 220 rows, but pred.txt contains 134,398 rows (excluding the header). I assume pred.bed contains high-confidence RBP binding sites from PAR-CLIP experiments, but I do not know what cutoff was used—for example, what criteria were applied to filter background—to derive the 220 sites from the 134,398 sites.

**Comment by santataRU** [comment]
The p-values in pred.txt seem to be Bonferroni corrected based on help. 

`omniCLIP run_omniCLIP --help     `   
usage: PROG run_omniCLIP [-h] --bg-dat DAT_FILE_BG --clip-dat DAT_FILE_CLIP
                         [--out-dir OUT_DIR] [--db-file GENE_ANNO_FILE]
                         [--bg-type {None,Coverage,Coverage_bck}]
                         [--max-it MAX_IT] [--tol-log-like TOL_LG_LIK]
                         [--nr_mix_comp NR_MIX_COMP] [--ign-diag] [--ign-GLM]
                         [--snp-ratio SNPS_THRESH]
                         [--snp-abs-cov SNPS_MIN_COV] [--norm_class]
                         [--seed RND_SEED]
                         [--diag_event_mod {DirchMult,DirchMultK}]
                         [--glm_weight GLM_WEIGHT] [--skip_diag_event_mdl]
                         [--pv PV_CUTOFF] [--emp-var] [--diag-bg]
                         [--fg_pen FG_PEN] [--filter-snps] [--no-subsample]
                         [--ign_out_rds] [--nb-cores NB_PROC] [--save-tmp]
                         [--tmp-dir TMP_DIR] [--verbosity VERBOSITY]
                         [--mask-ovrlp]

running the main omniCLIP program.

optional arguments:
  -h, --help            show this help message and exit
  --bg-type {None,Coverage,Coverage_bck}
                        Background type
  --max-it MAX_IT       Maximal number of iterations
  --tol-log-like TOL_LG_LIK
                        tolerance for lok-likelihood
  --nr_mix_comp NR_MIX_COMP
                        Number of diagnostic events mixture components
  --ign-diag            Ignore diagnostic event model for scoring
  --ign-GLM             Ignore GLM model for scoring
  --snp-ratio SNPS_THRESH
                        Ratio of reads showing the SNP
  --snp-abs-cov SNPS_MIN_COV
                        Absolute number of reads covering the SNP position
  --norm_class          Normalize class weights during glm fit
  --seed RND_SEED       Set a seed for the random number generators
  --diag_event_mod {DirchMult,DirchMultK}
                        Diagnostic event model
  --glm_weight GLM_WEIGHT
                        weight of the glm score with respect to the diagnostic
                        event score
  --skip_diag_event_mdl
                        Do not model the diagnostic events
  **--pv PV_CUTOFF        bonferroni corrected p-value cutoff for peaks in bed-file**
  --emp-var             use the empirical variance if it larger than the
                        expected variance
  --diag-bg             estimate diagnostic events for the background states
                        on the background
  --fg_pen FG_PEN       Penalty for fg during scoring
  --filter-snps         Do not fit diagnostic events at SNP-positions
  --no-subsample        Disable subsampling for parameter estimations
                        (Warning: Leads to slow estimation)
  --ign_out_rds         ignore reads where the ends map ouside of the genes
  --nb-cores NB_PROC    Number of cores o use
  --save-tmp            Safe temporary results
  --tmp-dir TMP_DIR     Output directory for temporary results
  --verbosity VERBOSITY
                        verbosity: 0 (default) gives information of current
                        state of site prediction, 1 gives aditional output on
                        runtime and meomry consupmtiona and 2 shows selected
                        internal variables
  --mask-ovrlp          Ignore overlapping gene regions for diagnostic event
                        model fitting

required arguments:
  --bg-dat DAT_FILE_BG  Path to the parsed background .dat file
  --clip-dat DAT_FILE_CLIP
                        Path to the parsed CLIP .dat file
  --out-dir OUT_DIR     Output directory for results
  --db-file GENE_ANNO_FILE
                        File where gene annotation is stored

**Comment by santataRU** [comment]
After I run parsingCLIP with `--collapsed --rev_strand 0`, with everything else the same, the number of peaks in the pred.bed increased from 220 rows to 27,437 rows. Using the CLIP data from `--collapsed --rev_strand 0` with background data (RNAseq dUTP stranded method) generated with `--rev_strand 0`, pred.bed output 20,443 rows, peaks make more sense in this case.
